src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -351,7 +351,7 @@ function getProjectDependencies(buildCommand: CodeTransformMavenBuildCommand, mo
             codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
             codeTransformMavenBuildCommand: buildCommand,
             result: MetadataResult.Fail,
-            reason: spawnResult.error ? spawnResult.error.message : spawnResult.stdout,
+            reason: 'CannotRunMavenShellCommand',
```

Comment:
Changing since `error` and `stdout` may contain PII and we want to log these via CW Logs anyway.